cmd/vcluster/cmd/start.go

```diff
@@ -90,15 +88,21 @@ func ExecuteStart(ctx context.Context, options *StartOptions) error {
 		return fmt.Errorf("license init: %w", err)
 	}
 
-	// check if there are existing vclusters in the current namespace. This can be achieved by listing the vcluster services
-	vclusterServices, err := listVclusterServices(ctx, vConfig.ControlPlaneClient, vConfig.ControlPlaneNamespace)
+	logger := log.GetInstance()
+	// check if there are existing vClusters in the current namespace
+	vClusters, err := find.ListVClusters(ctx, "", "", vConfig.ControlPlaneNamespace, logger)
 	if err != nil {
 		return err
 	}
-
-	logger := log.GetInstance()
-	// add a note for setting reuse-namespace config in v0.24 and a deprecation warning for multiple vcluster creation scenario
-	if len(vclusterServices) > 0 {
+	var vClusterExists bool
+	for _, v := range vClusters {
+		if v.Namespace == vConfig.ControlPlaneNamespace && v.Name != vClusterName {
```

Comment:
I see that we are already supplying vConfig.ControlPlaneNamespace to the find.ListVClusters function on line no. 93, so that it will limit the search to this namespace only. This makes the check for `v.Namespace == vConfig.ControlPlaneNamespace` redundant no?

Reply:
Yes I thought so too. There is a similar usage in CLI functionality ([reference](https://github.com/loft-sh/vcluster/blob/main/pkg/cli/create_helm.go#L137)). Since we're close to release, I leaned on the side of extra caution and chose to reuse it without any alteration (though it's bugging me as well!). 

Second reason is that, I'm working on the same part of code for next task as well so I'll test this part out with and without that namespace check and then remove it once I'm sure that it doesn't impact anything.